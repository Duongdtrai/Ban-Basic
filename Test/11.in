4
10
36253 20225 94860 21414 66038 2286 61524 50556 78785 74648 
6
25428 73380 43425 64530 10462 61974 
9
90314 6208 64889 47600 17877 23724 89929 65608 44790 
3
53798 41517 58927 
