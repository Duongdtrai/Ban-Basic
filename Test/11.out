1630047312816
37119077160
199744088780
4892944518
